extensions/wikia/ExactTargetUpdates/client/builders/DeleteRequestBuilder.class.php

```diff
@@ -30,6 +35,19 @@ public function build() {
 						[ Enum::USER_PROPERTIES_USER => $this->userId, Enum::USER_PROPERTIES_PROPERTY => $property ] );
 				}
 				break;
+			case self::WIKI_TYPE:
+				$objects = [ $this->prepareDataObject( Enum::CUSTOMER_KEY_WIKI_LIST,
+					[ Enum::WIKI_ID => $this->wikiId ] ) ];
+				break;
+			case self::WIKI_CAT_TYPE:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 9 to at most 5, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1151)